plugins/armorblox/icon_armorblox/triggers/get_incidents/trigger.py

```diff
@@ -0,0 +1,31 @@
+import insightconnect_plugin_runtime
+import time
+from datetime import datetime, timedelta
+from icon_armorblox.util.constants import ARMORBLOX_INCIDENT_API_TIME_FORMAT, ARMORBLOX_INCIDENT_API_TIME_DELTA_IN_DAYS
+from .schema import GetIncidentsInput, GetIncidentsOutput, Input, Output, Component
+# Custom imports below
+
+class GetIncidents(insightconnect_plugin_runtime.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='get_incidents',
+                description=Component.DESCRIPTION,
+                input=GetIncidentsInput(),
+                output=GetIncidentsOutput())
+
+    def run(self, params={}):
+        """Run the trigger"""
+        fetch_interval = params.get(Input.INTERVAL)
```

Comment:
Please specify the default interval value 
```suggestion
        fetch_interval = params.get(Input.INTERVAL, DEFAULT_INTERVAL_VALUE)
```